.github/workflows/issues.yaml

```diff
@@ -0,0 +1,11 @@
+name: Sync Issues to JIRA
+
+on:
+  issues:
+    types: [opened, reopened, closed]
+
+jobs:
+  update:
+    name: Update Issue
+    uses: canonical/sdcore-github-workflows/.github/workflows/issues.yaml@main
```

Comment:
This action requires `JIRA_URL` secret to be added in the project settings. Unfortunately I'd exactly remember the value. Perhaps Yazan does.

Reply:
Yes, we will store all high levels SD-Core modules here.

Reply:
Added.